lib/saluki-event/src/eventd/mod.rs

```diff
@@ -13,55 +37,57 @@ pub struct EventD {
     timestamp: u64,
     host: String,
     aggregation_key: String,
-    priority: EventPriority,
+    priority: Priority,
     source_type_name: String,
-    alert_type: EventAlertType,
+    alert_type: AlertType,
     tags: Vec<String>,
 }
 
 impl EventD {
-    /// Gets a reference to the title.
+    /// Returns the title of the event.
```

Comment:
This change in wording (like across the board for the fields in this type) is a little bit nit-picky, and I know the existing codebase has a lot of `Gets a reference to X`, but that's a pattern I want to clean up in the future.

In general, for accessors, this is the format I'd like to use for doc comments, since it better reflects the common wording for accessors, which is `Returns ...`.

We also do have some stuttering here -- the `of the event` suffix for each comment -- but this is generally the only where I think it's OK, because stuttering in types is always in your face since it's in the code, whereas in comments, it's hidden until you... look at the docs, or hover over a type and get intellisense, etc.

Not a hard and fast rule, but just trying to solidify the style here since we don't (yet) have a style guide.